articles/email/index.md

```diff
@@ -21,25 +22,25 @@ Users will see the sender's address in the **From Address** field when receiving
 
-* your emails will be sent from `no-reply@auth0.com` if you do not use a customized email template;
-* your emails will be sent from the email address of the first owner of your Auth0 account if you are using customized email templates.
+* Your emails will be sent from `no-reply@auth0.com` if you do not use a customized email template.
+* Your emails will be sent from the email address of the first owner of your Auth0 account if you are using customized email templates.
 
-**NOTE:** For security purposes, you may not send customized emails from any `@auth0.com` address. If you are an Appliance user, you may configure a similar domain blacklist.
+**NOTE:** For security purposes, you may not send customized emails from any `@auth0.com` address. If you are an appliance user, you may configure a similar domain blacklist.
 
 The **From Address** field supports the following macros:
 
 * `{application.name}`
 * `{connection.name}`
 
-You can use these macros to set the display name of the **From Address** to something that relates to the application for which the user signed up. For example, the field could display `{application.name} <support@fabrikamcorp.com>`, as opposed to `<support@fabrikamcorp.com>`.
+You can use these macros to set the display name of the **From Address** to something that relates to the application for which the user signed up. For example, the field could display `{application.name} <support@fabrikamcorp.com>`, as opposed to simply `<support@fabrikamcorp.com>`.
 
-You must add the [Sender Policy Framework (SPF)](http://en.wikipedia.org/wiki/Sender_Policy_Framework) and [DomainKeys Identified Mail (DKIM)](http://en.wikipedia.org/wiki/DKIM) DNS records, which allow Auth0 to send digitally-signed emails on your behalf. Without these records, the emails may end up in your users' junk folders. Additionally, your users may see the following as the **From Address**:
+You must add the [Sender Policy Framework (SPF)](http://en.wikipedia.org/wiki/Sender_Policy_Framework) and [DomainKeys Identified Mail (DKIM)](http://en.wikipedia.org/wiki/DKIM) DNS records to your domain's zone file to allow Auth0 to send digitally-signed emails on your behalf. Without these records, the emails may end up in your users' junkmail folders. Additionally, your users may see the following as the **From Address**:
 
 `MyApp support@mail128-21.atl41.mandrillapp.com on behalf of MyApp support@fabrikamcorp.com`
 
 #### SPF Configuration
 
-You can configure the SPF by adding a TXT record to your domain. You should set the host name to `@`, or leave it empty, depending on the provider. If this is the first time you are setting up an SPF record, you will need to declare it as follows:
+You can configure the SPF by adding a TXT record to your domain's zone file. You should set the host name to `@`, or leave it empty, depending on the provider. If this is the first time you are setting up an SPF record, you will need to declare it as follows:
```

Comment:
Im pretty sure it's the zone file that this TXT record is added to, please verify.